qa/tasks/cephfs/filesystem.py

```diff
@@ -823,7 +823,7 @@ def create(self, **kwargs):
                 assert(subvols['create'] > 0)
 
                 self.run_ceph_cmd('fs', 'subvolumegroup', 'create', self.name, 'qa')
-                subvol_options = self.fs_config.get('subvol_options', '')
+                svopts = self.fs_config.get('subvol_options', None)
```

Comment:
Acc to commit message we only want to replace `''` by `None`. Let's drop renaming from `subvol_options` to `svopts`. IOW, Let's keep it `subvol_options`, since renaming it is an unrelated and unnecessary change.

Reply:
ack

Reply:
I checked this commit again, in the code snippet above we still replace `subvol_options` by `svopts` (besides replacing `''` by `None`).

Reply:
I checked this commit again, in the code snippet above we still replace `subvol_options` by `svopts` (besides replacing `''` by `None`).